### PR TITLE
Testing Python 3.8 and PR # 969

### PR DESCRIPTION
**Pull request by DougBurke**
Let's see what Python 3.8 does to our "full" tests. We can see that without #967 the tests fail, but they pass after.

We should think about updating the Travis tests, but this is for after the 4.12.2 release.

This PR is not intended for merging. Just testing.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,19 +7,20 @@ jobs:
...
     - env: INSTALL_TYPE=develop TEST=none NUMPYVER="1.17" TRAVIS_PYTHON_VERSION="3.8"
-    # Full build (including ds9 and xspec), Python 3.5, setup.py develop, astropy, matplotlib 2
-    - env: XSPECVER="12.10.1n" NUMPYVER="1.11" FITS="astropy pytest-openfiles<=0.4.0 pytest-doctestplus<=0.5.0" INSTALL_TYPE=develop TEST=submodule MATPLOTLIBVER="2" TRAVIS_PYTHON_VERSION="3.5"
+    # Full build (including ds9 and xspec), Python 3.6, setup.py develop, astropy, matplotlib 2
     # As above, Python 3.6, setup.py install, xspec 12.10
     - env: XSPECVER="12.10.1n" NUMPYVER="1.11" FITS="astropy" INSTALL_TYPE=install TEST=submodule MATPLOTLIBVER="2" TRAVIS_PYTHON_VERSION="3.6"
     # As above, python 3.7, numpy 1.15, matplotlib 3
     - env: XSPECVER="12.10.1n" NUMPYVER="1.15" FITS="astropy" INSTALL_TYPE=install TEST=submodule MATPLOTLIBVER="3" TRAVIS_PYTHON_VERSION="3.7"
+    # As above, python 3.8, numpy 1.19, matplotlib 3
+    - env: XSPECVER="12.10.1n" NUMPYVER="1.19" FITS="astropy" INSTALL_TYPE=install TEST=submodule MATPLOTLIBVER="3" TRAVIS_PYTHON_VERSION="3.7"
```
Comment (anetasie) [inline .travis.yml:23]: @DougBurke Why do you have `TRAVIS_PYTHON_VERSION="3.7"` here? Should this be 3.8?

**Comment by anetasie** [comment]
@DougBurke I just made a comment.

File: .travis.yml (hunk 1 above)
Comment (DougBurke) [reply .travis.yml:23]: Doh. I'm less bothered about the linux 3.8 build though than the macOS one. This is just to have some evidence for a fix for whenever anyone gets around to fixing #730

**Comment by DougBurke** [comment]
So this looks promising (the only test that fails is macOS but that is a "barebones" test that is erroring out because of xvfbstuff, so test-setup related.

**Comment by DougBurke** [comment]
I've just added the latest update from #969 and have tried to fix the experimental macOS barebones build

**Comment by DougBurke** [comment]
Closed as #969 has now been merged.

**Pull request closed without merging**